lib/services/runtime/handlers/command.ts

```diff
@@ -9,6 +9,14 @@ import { F, T } from '@/lib/constants';
 import { IntentName, IntentRequest, RequestType } from '../types';
 import { mapSlots } from '../utils';
 
+const isPushCommand = (command: BaseNode.AnyCommonCommand): command is BaseNode.Command.Command & { diagram_id: string } => {
+  return !!(command as BaseNode.Command.Command).diagram_id;
```

Comment:
the important thing here is that `Command.Command` uses `diagram_id` while `Intent.Command` is `diagramID?`